djangoevents/__init__.py

```diff
@@ -19,9 +21,20 @@
     'entity_mutator',
     'singledispatch',
     'publish',
+    'store_event'
     'subscribe',
     'unsubscribe',
     'subscribe_to',
     'BaseEntity',
     'EventSourcingWithDjango'
 ]
+
+
+def publish(event):
+    warnings.warn("`publish` is depreciated. Please switch to: `store_event`", DeprecationWarning)
+    return es_publish(event)
+
+
+def store_event(event):
```

Comment:
@renou  mainly because previous naming was very misleading. I'd expect that "publish"
method would broadcast my event across the system. And djangoevents simply doesn't support
this - by default "publish" only stores the event in repository. We can discuss this offline
but I'd say that being straightforward is more important than flexibility here (it's hard to
reason about API if "publish" essentially means "store event").

Moreover, we should probably discourage usage of eventsourcing's subscribe functions. This is
exactly why I wanted to hide eventsourcing and provide own abstractions in the first place.

Btw, @piotrekno1, we should also update README, I've forgotten about it (we use "publish" there).